src/main/java/seedu/address/model/module/exam/Exam.java

```diff
@@ -24,11 +24,11 @@
     /**
      * Constructs a {@code Exam}
      *
-     * @param name Name of exam
-     * @param date Date of exam
+     * @param name     Name of exam
+     * @param date     Date of exam
      * @param timeslot Timeslot of the exam
-     * @param venue Optional venue for the exam
-     * @param link Optional link for the exam
+     * @param venue    Optional venue for the exam
+     * @param link     Optional link for the exam
```

Comment:
any reason for these changes?

Reply:
Just for consistent format. Would it be better to revert?

Reply:
oh i see! nah it looks good, couldnt tell from the diff the alignment haha